.changeset/fair-mice-shave.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
this empty changesets file can be now safely removed, since you've added a meaningful changeset message